gpMgmt/bin/gppylib/logfilter.py

```diff
@@ -273,6 +273,17 @@ def next(self):
         #we need to make a minor format change to the log level field so that
         # our single regex will match both.
         item[16] = item[16] + ": "
+
+        # There is a text containing \n or | symbols. We need to add quotation 
+        # marks, for parsers to process csv text correctly. We also need to 
+        # double quote in places where they already are.
```

Comment:
1. There are trailing spaces in lines of comment. Take care of it. You might setup necessary plugin to your editor or check it via `git diff` before commiting patch 
2. Rewrite comment that we have to quote such column values that contain new line, csv delimiter or double quotation marks. The last ones have to be escaped by another double quote. All this have to be done to generate valid csv.